src/languages/en.ts

```diff
@@ -2293,6 +2293,28 @@ export default {
                             return 'Updating people list';
                         case 'quickbooksOnlineSyncApplyClassesLocations':
                             return 'Updating report fields';
+                        case 'xeroSyncImportChartOfAccounts':
```

Comment:
@lakchote we need to add all the translations in spanish too. Or the build would break

Reply:
Yes! I thought about it at first and then because of having to swich tasks, forgot about it.

I've asked for spanish translations and will update PR as soon as I have them.

Reply:
Asked for spanish translations [here](https://expensify.slack.com/archives/C21FRDWCV/p1714721884184949) for reference (for internal employees)